lib/request-options.js

```diff
@@ -33,7 +33,9 @@ module.exports = (libOptions, logger) => {
             resolveWithFullResponse: true
         };
 
-        options.timeout = libOptions.timeoutDuration;
+        if (libOptions.timeoutDuration) {
```

Comment:
I think this will always exist. It's set in lib/index.js. I guess it never hurts to check though.